front/app/modules/commercial/posthog_integration/index.ts

```diff
@@ -37,8 +37,8 @@ const initializePosthog = async (token, user, appConfig) => {
   posthog.init(token, {
     api_host: 'https://eu.posthog.com',
     autocapture: false,
-    persistence: 'memory', //no cookies
-    loaded: function (ph) {
+    persistence: 'memory', // no cookies
+    loaded(ph) {
```

Comment:
Do we need this change for the folders or is fixing something else?

Reply:
It was changed automatically by the linter or something.